doc/changelog/01-kernel/09867-floats.rst

```diff
@@ -0,0 +1,5 @@
+- Add primitive floats implementing binary64 floating-point arithmetic,
+  see :ref:`primitive-floats`
```

Comment:
IMHO this is an achievement that deserves a bit more showing-off, i.e. a tag longer changelog entry.